src/components/select/select.jsx

```diff
@@ -173,48 +173,84 @@ const Select = forwardRef(function Select(props, ref) {
 
   return (
     <div ref={wrapperRef} className={props.className}>
-      <AbstractCustomField
```

Comment:
I'm really glad to see any mention of the words `custom field` ripped out of a component that has nothing to do with custom fields